python/ray/autoscaler/gcp/example-full.yaml

```diff
@@ -109,6 +109,9 @@ file_mounts: {
 #    "/path2/on/remote/machine": "/path2/on/local/machine",
 }
 
+# Whether changes to mounted files in the head node should sync to the worker node continuously
```

Comment:
# Whether changes to mounted files in the head node should sync to the worker nodes continuously.